types/eslint-scope/index.d.mts

```diff
@@ -0,0 +1,27 @@
+import {
```

Comment:
Do we need both `index.d.ts` and `index.d.mts`? They're the same, right?

Reply:
Having a `index.d.ts` as "default" should provide higher-incorp for old node / ts version which does not know about how to resolve `index.d.mts`.